pkg/controller/remotemachineset/gcpactuator.go

```diff
@@ -142,3 +168,126 @@ func (a *GCPActuator) getImageID(cd *hivev1.ClusterDeployment, logger log.FieldL
 		return "", errors.New(msg)
 	}
 }
+
+// obtainLease uses the Hive MachinePoolNameLease resource to obtain a unique, single character
+// for use in the name of the machine pool. We are severely restricted on name lengths on GCP
+// and effectively have one character of flexibility with the naming convention originating in
+// the installer.
+func (a *GCPActuator) obtainLease(pool *hivev1.MachinePool, logger log.FieldLogger) (rune, error) {
```

Comment:
Return a `string` rather than a `rune`. Returning a `rune` (1) unnecessarily limits what this function can do in the future and (2) provides no benefit over a `string` as it will be converted to a `string` by the caller.